iceoryx_dds/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.7)
 
-set(iceoryx_dds_gateway_VERSION 0.1.0)
-project(iceoryx_dds_gateway VERSION ${iceoryx_dds_gateway_VERSION})
+set(iceoryx_dds_VERSION 0.1.0)
+project(iceoryx_dds VERSION ${iceoryx_dds_VERSION})
```

Comment:
No I don't think so, we should have completely different and independent versioning system here. It should be hexadecimal but with roman numbers. And of course 5 levels mixed with letters, like XI.blubb.12.ae. ;) What do you think @elBoberido @FerdinandSpitzschnueffler 

Or we have a consistent version number throughout the project ... this would also be fine by me!

Reply:
:heavy_check_mark: